src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -963,11 +963,12 @@ private InstructorFeedbackResultsQuestionTable buildQuestionTableAndResponseRows
             }
             
             // If question specific sorting is not needed, responses are sorted
-            // by default order (first by team name, then by display name)
+            // in the following order (first by recipient name, then by team name,
+            // then by display name)
             if (questionDetails.isQuestionSpecificSortingRequired()) {
                 Collections.sort(responseRows, questionDetails.getResponseRowsSortOrder());
             } else {
-                responseRows = InstructorFeedbackResultsResponseRow.sortListWithDefaultOrder(responseRows);
+                responseRows = InstructorFeedbackResultsResponseRow.sortListByRecipientNameThenDefaultOrder(responseRows);
```

Comment:
@swxsw, I'll try to verify if the issue is still valid in the current version of code soon, but this doesn't look like that right solution because the issue is about the statistics table (stuff like 'Average', etc..), but the code here affects the display of all individual responses. In addition, the code here affects the display logic for all question types, not just 'constant sum' question.